cypress/helpers/cypressTasks/cognito/cognito-helpers.ts

```diff
@@ -61,20 +70,44 @@ const getUserPoolId = async (): Promise<string> => {
   return userPoolId;
 };
 
-export const getCognitoUserIdByEmail = async (
-  email: string,
-): Promise<string> => {
-  const userPoolId = await getUserPoolId();
-  const foundUser = await getCognito().adminGetUser({
+export const createAccount = async ({
+  isIrsEnv,
+  password,
+  role,
+  userName,
+}: {
+  userName: string;
+  password: string;
+  role: string;
+  isIrsEnv: boolean;
+}): Promise<string> => {
+  const userPoolId = await getUserPoolId(isIrsEnv);
+  await getCognito().adminCreateUser({
+    TemporaryPassword: password,
+    UserAttributes: [
+      {
+        Name: 'custom:role',
+        Value: role,
+      },
+      {
+        Name: 'email',
+        Value: userName,
+      },
+      {
+        Name: 'email_verified',
+        Value: 'true',
+      },
+    ],
     UserPoolId: userPoolId,
-    Username: email.toLowerCase(),
+    Username: userName,
   });
-
-  const userId = foundUser.UserAttributes?.find(
-    element => element.Name === 'custom:userId',
-  )?.Value!;
-
-  return userId;
+  await getCognito().adminSetUserPassword({
+    Password: password,
+    Permanent: true,
+    UserPoolId: userPoolId,
+    Username: userName,
+  });
+  return `created user ${userName}`;
```

Comment:
Is this consumed anywhere?

Reply:
it's not, but cypress tasks need to return *something*. would it be better to consume it in an assert? maybe change the return to a void?

Reply:
I believe the tasks can also return null!